boards/cats_rev1Pro/src/tasks/task_init.c

```diff
@@ -121,10 +121,11 @@ _Noreturn void task_init(__attribute__((unused)) void *argument) {
 
   osDelay(10);
 
-  /* Check if the FSM configurations make sense */
-  if (global_cats_config.config.control_settings.liftoff_acc_threshold < 1500) {
-    log_error("Acceleration Threshold is not configured Properly!");
```

Comment:
are we somehwere making sure that the user cannot set an acceleration threshold lower than 3500? What happens if the user configures the board over USB to have an acceleration threshold of 500?